server/report_builder/README.md

```diff
@@ -101,7 +101,24 @@ For example, to print a report for a stocktake with id "d734fd45-064e-4ddd-9886-
 ```
 
 
-## References to other template definitions (not supported in the report builder yet) 
+## References to other template definitions
 It's possible to refer to other template resources that already exist on the server, e.g. to refer to a common headers or icons.
 
-Please request if needed...
+To refer to a resource from an existing report add a reference file like `icon1.ref.json` to the project with the content:
```

Comment:
Awesome, I've lost some context about how reporting works, does output.json have these refs imbedded ?

Reply:
Yes the report definition file (e.g. output.json) contains a list of entries. One entry type is a reference to an entry in other report definitions. So this was already implemented but there was no way to specify this in the report builder.

Reply:
Cool, so it would be like a reference report row ?

Reply:
yes and this reference row/entry gets resolved to the real value before it is passed into the Tera engine